deployment/roles/haystack/tasks/main.yml

```diff
@@ -0,0 +1,73 @@
+---
+- name: "{{ client }} | Installing haystack"
+  ansible.builtin.debug:
+    msg: Haystack will be installed on port {{ deployments[client].haystack_port }}
+- name: "{{ client }} | Setting client installation folder"
+  ansible.builtin.set_fact:
+    client_installation_directory: "{{ installation_directory }}/haystack_{{ client }}"
+- name: "{{ client }} | clone haystack"
+  ansible.builtin.git:
+    repo: 'https://github.com/deepset-ai/haystack.git'
+    dest: "{{ client_installation_directory }}"
+    version: "{{ haystack_commit }}"
+    update: no
+  register: haystack
+#  We are needing that since we are modifying files in the git repo below
+- name: "{{ client }} | force clone haystack"
+  ansible.builtin.git:
+    repo: 'https://github.com/deepset-ai/haystack.git'
+    dest: "{{ client_installation_directory }}"
+    version: "{{ haystack_commit }}"
+    force: yes
+  when: not haystack.after.startswith(haystack_commit)
+- name: "{{ client }} | change group ownership (https://github.com/ansible/ansible/issues/51601)"
+  file:
+    path: "{{ client_installation_directory }}/haystack" # this should be as same as `dest` above
+    group: "piaf-deployment"
```

Comment:
in the README we mention the group `piaf` : are we refering to the same group as `piaf-deployment` ? if yes better to update the README right ?

Reply:
Well spotted